### PR TITLE
Check TTL, stale check result monitoring

### PR DESCRIPTION
**Pull request by portertech**
https://trello.com/c/zluCic91/23-as-an-operator-i-would-like-to-detect-stale-check-executions-ttl

### CODE REVIEW

**Comment by portertech** [comment]
Ready for review :+1:

File: lib/sensu/server/process.rb [hunk 1]
```diff
@@ -644,6 +644,50 @@ def setup_client_monitor
...
+            @redis.smembers("result:#{client_name}") do |checks|
+              checks.each do |check_name|
+                result_key = "#{client_name}:#{check_name}"
+                @redis.get("result:#{result_key}") do |result_json|
+                  unless result_json.nil?
+                    check = MultiJson.load(result_json)
+                    next unless check[:ttl] && check[:executed]
+                    time_since_last_execution = Time.now.to_i - check[:executed]
+                    if time_since_last_execution >= check[:ttl]
+                      check[:output] = "Last check execution was "
```
Comment (portertech) [inline lib/sensu/server/process.rb]: Bonus of building on the stored result: check handler and custom attributes etc.

**Comment by amdprophet** [comment]
:+1: 

giphy-4

**Comment by roobert** [comment]
jprse

**Comment by portertech** [comment]
@roobert always the good horse head gifs :+1:

**Pull request merged**